Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:18
+MAINTAINER SDF Wallets Team <wallet-eng@stellar.org>
+
+RUN mkdir -p /app
+WORKDIR /app
```

Comment:
do we need to install node and yarn and stuff here? example: https://github.com/stellar/laboratory/blob/master/Dockerfile#L11

Reply:
nope, using the node image so they come built in.

Reply:
Ah v cool 👍